script.js

```diff
@@ -30,3 +30,249 @@ function menuDisappear() {
+      </div>
+      <div class="project-info">
+        <h3 class="project-title">${workHistory[0].cardTitle}</h3>
+        <div>
+          <ul class="project-tags">
+            <li>${workHistory[0].cardTags[0]}</li>
+            <li>${workHistory[0].cardTags[1]}</li>
+            <li>${workHistory[0].cardTags[2]}</li>
+            <li>${workHistory[0].cardTags[3]}</li>
+          </ul>
+        </div>
+        <div class="btn-container">
+          <button id="5" type="submit" class="btn-all">See project</button>
+        </div>
+      </div>
+    </div>
+
+  </div>`;
+
+workContainer.appendChild(sectionDiv);
+
+//  ----Mobile Popup Window ----
+
+const projectBtn = document.querySelectorAll('.work-history .btn-all');
+for (let i = 0; i < projectBtn.length; i += 1) {
+  projectBtn[i].addEventListener('click', () => {
+    const projects = [
+      {
+        projectName: 'Keeping track of hundreds of components',
+        projectDescription: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s.',
+        projTechno: ['Ruby on Rails', 'CSS', 'JavaScript'],
+        linkToLive: 'https://nahid1911.github.io',
+        linkToSource: 'https://github.com/Nahid1911/Nahid-s_Portfolio',
+      },
+      {
+        projectName: 'Keeping track of hundreds of components',
+        projectDescription: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s.',
+        projTechno: ['JavaScript', 'CSS', 'Ruby on Rails'],
+        linkToLive: 'https://nahid1911.github.io',
+        linkToSource: 'https://github.com/Nahid1911/Nahid-s_Portfolio',
+      },
+
+      {
+        projectName: 'Keeping track of hundreds of components',
+        projectDescription: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s.',
+        projTechno: ['Ruby on Rails', 'CSS', 'JavaScript'],
+        linkToLive: 'https://nahid1911.github.io',
+        linkToSource: 'https://github.com/Nahid1911/Nahid-s_Portfolio',
+      },
+
+      {
+        projectName: 'Keeping track of hundreds of components',
+        projectDescription: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s.',
+        projTechno: ['Ruby on Rails', 'CSS', 'JavaScript'],
+        linkToLive: 'https://nahid1911.github.io',
+        linkToSource: 'https://github.com/Nahid1911/Nahid-s_Portfolio',
+      },
+
```

Comment:
[OPTIONAL]
- [ ] Since the variable `const projects` has a constant and static value, Kindly move it outside of the for loop for better performence.